content/Improve your Rails App searches with Elasticsearch - Leandro Dominguez.md

```diff
@@ -0,0 +1,216 @@
+# Improve your Rails Apps searches with Elasticsearch
```

Comment:
Suggested Title Changes:  How to integrate Elasticsearch in your Ruby on Rails App (Example Tutorial)   

OR 

How to Build Application Search with Ruby on Rails and Elastic (Example Tutorial)

Reply:
I wrote you via Slack to talk a bit about this.